content/courses/tokens-and-nfts/nfts-with-metaplex.md

```diff
@@ -69,75 +69,106 @@ program.
   offchain component of NFT metadata.
 
 In the following sections, we'll cover the basics of using the
-`metaplex-foundation/js` npm module (also known as the Metaplex JavaScript SDK)
-to prepare assets, create NFTs, update NFTs, and associate an NFT with a broader
-collection. For more information on `metaplex-foundation/js` see the
-[Metaplex JavaScript SDK README](https://github.com/metaplex-foundation/js) and
-the
-[Metaplex JS SDK Examples](https://github.com/metaplex-foundation/js-examples).
-
-<Callout type="note">The Metaplex Foundation has newer mechanisms to create
-NFTs, including [Umi](https://github.com/metaplex-foundation/umi) and
-[MPL Core](https://github.com/metaplex-foundation/mpl-core/blob/main/clients/js/README.md)
-but [Metaplex JavaScript SDK](https://github.com/metaplex-foundation/js) remains
-the simplest and most popular JS/TS client to create NFTs with broad wallet
-support.</Callout>
-
-#### Metaplex instance
-
-A `Metaplex` instance serves as the entry point for accessing the Metaplex SDK.
-This instance accepts a connection used to communicate with the cluster.
-Additionally, developers can customize the SDK's interactions by specifying an
-"Identity Driver" and a "Storage Driver".
-
-The Identity Driver is a keypair that can be used to sign transactions, which is
-a requirement when creating an NFT. The Storage Driver is used to specify the
-storage service you want to use for uploading assets. The `irysStorage` driver
-is the default option, and it uploads assets to Irys, a permanent and
-decentralized storage service.
-
-Below is an example of how you can set up the `Metaplex` instance for devnet.
+`metaplex-foundation/token-metadata` plugin with UMI to prepare assets, create
+NFTs, update NFTs, and associate an NFT with a broader collection. For more
+information on `metaplex-foundation/token-metadata` see the
+[developer docs for Token Metadata](https://developers.metaplex.com/token-metadata).
+
+<Callout type="note">The Metaplex Foundation released a new standard for
+creating and managing NFTs called
+[Metaplex Core](https://developers.metaplex.com/core), that utilizes a single
+account model. But the only way to add metadata to SPL tokens is still done by
+using the Metadata program. </Callout>
```

Comment:
```suggestion
creating and managing NFTs called
[Metaplex Core](https://developers.metaplex.com/core), where an NFT simply has an owner, with the owner changing when the NFT is transferred. However the most common style of NFT is still by making a Solana SPL token with some Metadata attached via the Metaplex Metadata program, so that's what we'll be using in this tutorial.</Callout>
```

Can you simplify this section? Trying to avoid use of vague terms like 'account model' and mention specific differences. This is one suggestion .